DebuggingUtils syntax error for new projects

<!-- Please search existing issues for potential duplicates before filing yours:
https://github.com/godotengine/godot/issues?q=is%3Aissue
-->

**Godot version:**
Godot_v3.0-rc1_mono_win64


**OS/device including version:**
Windows XP


**Issue description:**
Build fails if my project includes a c# script.


**Steps to reproduce:**
I have Mono ~~4.5.0~~ 5.4.0 and Microsoft Build Tools 2015 installed (these were working with 3.0 beta 2)

I have deleted my `C:\Users\<username>\AppData\Roaming\Godot` directory.

If I create a new project and attached a new c# script to a node then when I build I receive three errors:

    Core\DebuggingUtils.cs(37,42): error CS1026: ) expected 
    Core\DebuggingUtils.cs(37,52): error CS1002: ; expected 
    Core\DebuggingUtils.cs(37,52): error CS1513: } expected 

My `C:\Users\<username>\AppData\Roaming\Godot\mono\solutions\GodotSharp_-<somenumber>\Core\DebuggingUtils.cs` contains the following lines:

    // Lines 37-38
    if (methodBase is MethodInfo methodInfo)
        sb.AppendTypeName(methodInfo.ReturnType);

If I modify them to this then no further errors occur:

    // Lines 37-38
    if (methodBase is MethodInfo)
        sb.AppendTypeName(((MethodInfo)methodBase).ReturnType);


**Minimal reproduction project:**
Attached zip contains example project and copy of my `C:\Users\<username>\AppData\Roaming\Godot\mono` directory

[DebuggingUtils_test_files.zip](https://github.com/godotengine/godot/files/1631642/DebuggingUtils_test_files.zip)

--- a/modules/mono/glue/cs_files/DebuggingUtils.cs
+++ b/modules/mono/glue/cs_files/DebuggingUtils.cs
@@ -34,8 +34,8 @@ namespace Godot
 
             StringBuilder sb = new StringBuilder();
 
-            if (methodBase is MethodInfo methodInfo)
-                sb.AppendTypeName(methodInfo.ReturnType);
+            if (methodBase is MethodInfo)
+                sb.AppendTypeName(((MethodInfo)methodBase).ReturnType);
 
             sb.Append(methodBase.DeclaringType.FullName);
             sb.Append(".");

